srcs/utils/error_exit.c

```diff
@@ -30,7 +30,7 @@ void	error_message(t_error code)
 	if (code == ERR_NO_PRINT)
 		return ;
 	ft_putstr_fd(""LIGHT_RED"Error \n"END_COLOR"", STDERR_FILENO);
-	if (code == WRONG_ARGC)
+	else if (code == WRONG_ARGC)

```

Comment:
actually no, it shouldn't be else if, because the previous line (when we're printing Error in red) is not under the if (nor should it be). And since we are returning in the previous if it's fine, no need for else

Reply:
oh shit yeah, I changed that for a TERMCAP error and then realised we don't need it and forgot to change it back ! haha